github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRLabelExistsEvent/config.groovy

```diff
@@ -2,7 +2,7 @@ package org.jenkinsci.plugins.github.pullrequest.events.impl.GitHubPRLabelExists
 
 def f = namespace(lib.FormTagLib);
 
-f.entry(field: "skip", title: "Skip PR?") {
+f.entry(field: "skip", title: "Skip pull requests with label(s)?") {
```

Comment:
Given that this goes infinite if you're not skipping, might it make more sense to get rid of the possibility of triggering at all and just use it for skipping?